Story/Episode 3/np_010851.csv

```diff
@@ -0,0 +1,53 @@
+text01007#0,"""Over the years, the objectives of ARKS seemed to have veered off course. Somehow, I managed to guide it back. """
+11_voice_NPC_5085_100#0,"""Exploring the Universe, Investigate Planets. Encountering any Darkers, fight and defeat them. That is ARKS."""
+11_voice_NPC_5085_106#0,"""About the Oracle huh? Oracle is basically an enormous fleet of Ships."""
+text02002#0,"""Each ARKS Ship serves as a base of operations for ARKS and also as the residence for its citizens."""
+text02003#0,"""The fleet is made up of such ships."""
+text02004#0,"""Currently, we are in just one of the many Ships. Just a section of this Ship. """
+text02005#0,"""And the ship at the heart of the fleet, the \""Mothership\"", houses the core of my existence."""
+11_voice_NPC_5085_108#0,"""Previously, Xion was the core of the Mothership. However, we lost her to [Loser]. """
+text02007#0,"""Since then, I have succeeded Xion, assuming control of the whole fleet."""
+text02008#0,"""That sounds confusing huh.. To put it simply, Oracle is under my control."""
+text02009#0,"""Though it might get lonely without Xion... I have to push on."""
+11_voice_NPC_5085_100#1,"""For the future of Oracle. For ARKS. For you and me."""
+text03001#0,"""About Darkers... They spread throughout the Universe. An enemy of the Universe, an enemy of every living being."""
+text03002#0,"""Darkers consume all in their path. Every civilization, every living thing."""
+text03003#0,"""Only ARKS can permanently defeat Darkers with the use of Photons."""
+text03004#0,"""Even if someone is able to defeat the Darkers without the use of Photons, the residual fragments will eventually corrupt him. """
+11_voice_NPC_5085_105#0,"""Under the Darkers corruption, they lose any consciousness of themselves, mutating into a ferocious enemy."""
+text03006#0,"""This is not only limited to organic beings. Mechanical beings and Artificial Intelligence are equally susceptible. """
+text03007#0,"""There are variances in mutation, however they end up corrupted all the same. Therefore, only ARKS is able to defeat Darkers."""
+11_voice_NPC_5085_104#1,"""Other means of defeating them may exist, if you consider the endless possibilities of the Universe."""
+text03009#0,"""For now, only ARKS is able to exterminate the Darkers. Which is why we are travelling across the Universe."""
+11_voice_NPC_5085_104#2,"""Dark Falz... To be honest, I do not have much information regarding them... So this will be short."""
+text04002#0,"""Currently, there is 5 confirmed existence of Dark Falz. [Elder], [Apprentice], [Loser], [Double] and lastly, [Persona]"""
+text04003#0,"""I can't say much about their characteristics... Well, you can simply think of them as the Commanders of Darkers."""
+text04004#0,"""[Elder] and [Apprentice] is more active than others in attacking ARKS."""
+text04005#0,"""[Loser] was defeated by Regius.. Due to [Double]'s power however, clones of [Loser] appear frequently."""
```

Comment:
"[Loser] was defeated by Regius, but was devoured by [Double], and so clones of it appear frequently."